### PR TITLE
added web visualization of multiple data reports

### PR DESCRIPTION
**Pull request by DavidSarlle**
This branch implements the possibility to select multiple saved reports to visualize data in the same webpage using qq-live. Unlike qq-compare, this function will show multiple saved reports without checking if the selected reports are comparable. 

Checklist:
- [ ] Reviewers confirm new code works as expected.
- [ ] Tests are passing.
- [ ] Coverage does not decrease.
- [ ] Documentation is updated.

### CODE REVIEW

File: src/qibocal/calibrations/characterization/resonator_spectroscopy.py [hunk 1]
```diff
@@ -135,7 +136,7 @@ def resonator_spectroscopy(
 
 
 @plot("Frequency vs Attenuation", plots.frequency_attenuation_msr_phase)
-@plot("MSR vs Frequency", plots.frequency_attenuation_msr_phase__cut)
+# @plot("MSR vs Frequency", plots.frequency_attenuation_msr_phase__cut)
```
Comment (scarrazza) [inline src/qibocal/calibrations/characterization/resonator_spectroscopy.py:139]: Are you removing this line for debugging?

File: src/qibocal/calibrations/characterization/qubit_spectroscopy.py [hunk 2]
```diff
@@ -8,6 +9,8 @@
 from qibocal.decorators import plot
 from qibocal.fitting.methods import lorentzian_fit
 
+# from qibocal.calibrations.characterization.utils import get_polycoef_flux
+
```
Comment (scarrazza) [inline src/qibocal/calibrations/characterization/qubit_spectroscopy.py:13]: What about this?

**Review by andrea-pasquale — COMMENTED** [review]
Thanks for implementing this @DavidSarlle.
You can find below a few comments.

A possible suggestion maybe not related to this PR. When the `qq-compare` command is failing it would be cool to show the error on the same page instead of showing it in a new web page:
Screenshot from 2022-12-09 10-35-34

File: src/qibocal/calibrations/characterization/qubit_spectroscopy.py (hunk 2 above)
Comment (DavidSarlle) [reply src/qibocal/calibrations/characterization/qubit_spectroscopy.py:13]: I have removed this line. It is not used in any place in the qubit spectrosocopy. It was introduced by @maxhant at some point to be used in one of his branchs.

File: src/qibocal/calibrations/characterization/resonator_spectroscopy.py (hunk 1 above)
Comment (DavidSarlle) [reply src/qibocal/calibrations/characterization/resonator_spectroscopy.py:139]: Uncommented.

File: src/qibocal/calibrations/characterization/qubit_spectroscopy.py (hunk 2 above)
Comment (DavidSarlle) [reply src/qibocal/calibrations/characterization/qubit_spectroscopy.py:13]: The method was used to get the resonator polycoef flux matrix used in qubit spectroscopy track and sstored in the runcard. But The matrix was obsolete and the values were calculated in a different way. So I have calculated them and stored from the resonator_spectroscopy_method and we can access as any othe parameter stored in the runcard. So the method has been removed and it is not needed anymore.

File: src/qibocal/calibrations/characterization/resonator_spectroscopy.py [hunk 3]
```diff
@@ -228,9 +228,9 @@ def resonator_spectroscopy_flux(
     frequency_range = (
         np.arange(-freq_width, freq_width, freq_step) + resonator_frequency
     )
-    current_range = (
-        np.arange(current_min, current_max, current_step) + qubit_biasing_current
-    )
+    current_range = np.arange(
+        current_min, current_max, current_step
+    )  # + qubit_biasing_current
```
[the remark this answers is not in the document]
Comment (DavidSarlle) [reply src/qibocal/calibrations/characterization/resonator_spectroscopy.py]: I did it for debugging. Now is active again.

File: src/qibocal/web/server.py [hunk 4]
```diff
@@ -21,36 +21,59 @@ def page(path=None):
...
+            command = "qq-compare " + list_of_folders
+            try:
+                command_output = subprocess.check_output([command], shell=True)
+
+            except subprocess.CalledProcessError as e:
+                return "An error occurred while trying to generate qq-report.<br>Please check folder compatility."
+
+            return redirect("/data/qq-compare", code=302)
+
+        if request.form["qq-compare"] == "combine":
```
[the remark this answers is not in the document]
Comment (DavidSarlle) [reply src/qibocal/web/server.py:59]: Done :)

**Comment by DavidSarlle** [comment]
> Thanks for implementing this @DavidSarlle. You can find below a few comments.
> 
> A possible suggestion maybe not related to this PR. When the `qq-compare` command is failing it would be cool to show the error on the same page instead of showing it in a new web page: Screenshot from 2022-12-09 10-35-34

Implemented

**Review by andrea-pasquale — APPROVED** [review]
Thanks for the updates @DavidSarlle.
This PR is ready to merge in my opinion.

**Pull request merged**